np.py

```diff
@@ -80,9 +80,13 @@ def main() -> None:
 
     if not args.loop:
         now_playing = lastfm_network.get_user(args.username).get_now_playing()
-        output(is_track_loved(now_playing))
-        if args.say:
-            say(now_playing)
+        if type(now_playing) is pylast.Track:
+            output(is_track_loved(now_playing))
+            if args.say:
+                say(now_playing)
+        else:
+            output("No track currently playing")
+
```

Comment:
I actually have this locally but not yet committed, how does this look:
```suggestion
        if not now_playing:
            return
        output(is_track_loved(now_playing))
```

Reply:
Thanks for reviewing.  Seems fine to me and much more succinct. :)